src/Network/Network.Management.Sdk/README.md

```diff
@@ -28,7 +28,9 @@ use-extension:
 ###
 ``` yaml
 commit: a6237cbef545ae4cd0d3674a7cb3effa2b7b13d8
+commit1: 7bad2d7844e5346fb23628c2ef8fc96fbe9b1549
 input-file:
+  - https://github.com/Azure/azure-rest-api-specs/blob/$(commit1)/specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/applicationGateway.json
```

Comment:
All swagger files should be under the same commit. If you want to use a new commit id, please replace the value of the commit instead of creating a new variable. After that you need to re-generate the SDK and check in all the changes. Partial update will result in CI failure.

Reply:
removed commit1 and used autorest --use:@autorest/powershell@4.x to re-generate SDK, but many generated changes(1439 files generated) are not related to my update. Do I need to add all those changes?

Reply:
re-generated SDK and add all auto reset changes.